web/film-club-client/src/components/FilmDetailPage.js

```diff
@@ -1,54 +1,193 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
-import { Box, Image, Heading, Text } from 'grommet';
+import { Box, Image, Heading, Layer, Text } from 'grommet';
+import { CirclePlay } from 'grommet-icons';
 
 import Header from './Header';
 
-const FilmDetailPage = props => {
+const FilmDetailPage = (props) => {
+	const [movieDetails, setMovieDetails] = useState({});
+	const [showVideo, setShowVideo] = useState(false);
+
 	// React Router URL Parameter object
 	let { id } = useParams();
 
-	// const getMovieDetails = async (id) => {
-	//     const response = await fetch(`http://localhost:8080/v1/films/${id}`)
-	//     const data = await response
-	//     // props.setMovieId(data)
-	//     console.log(data)
-	// }
-
-	// getMovieDetails(id)
+	// // Fetch all movie details with {id} from params, on component load
+	useEffect(() => {
+		// Use the {id} to fetch details about the corresponding movie
+		const getMovieDetails = async (id) => {
+			const response = await fetch(`http://localhost:8080/v1/films/${id}`);
```

Comment:
At some point we should figure out a good `baseUrl` strategy because `localhost` is already showing up in a lot of places.